components/entitlement/org.wso2.carbon.identity.entitlement/src/main/java/org/wso2/carbon/identity/entitlement/internal/EntitlementServiceComponent.java

```diff
@@ -320,10 +320,10 @@ protected void activate(ComponentContext ctxt) {
     /**
      * Adds policy files with unique policyIDs to the registry.
      *
-     * @param policyIdList
-     * @param fileList
-     * @return
-     * @throws IOException
+     * @param policyIdList List of IDs of existing policies.
```

Comment:
Shall we align the descriptions?

Reply:
Fixed in 3e3ba58611196513cf53893ed999e89cefee935e